works.html

```diff
@@ -71,7 +71,300 @@
         </div>
       </header>
       <main class="main">
-        <!-- Start your work here ... -->
+        <div class="main__global-container">
+          <div class="main__global-container-title">
+            <h2 class="title-heading">Буровые работы</h2>
```

Comment:
Кстати, у тебя на странице нет ни одного h1. Почему бы не сделать этот заголовок?
```suggestion
            <h1 class="main__title-heading">Буровые работы</h1>
```